elastic/apm-server
# feat: support standalone APM Server with Tilt

## Motivation/summary

Update Tilt to use standalone APM Server and remove elastic-agent and fleet dependency.

APM Server is run with debug logging and elasticsearch output configured.
The producing/packaging Dockerfile is used to build the image.

## Checklist



~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~



## How to test these changes

- tilt up
- `curl -X POST -H "Content-Type: application/x-ndjson" localhost:8200/intake/v2/events?verbose=true --data-binary @./testdata/intake-v2/errors.ndjson`
- open kibana (localhost:5601) and look at the `logs-*` data view.
- you should see the events

NOTE: mock events from in repo have a really old timestamp so you might need to adjust the time range (the suggestion is to just click show all).

## Related issues

Closes https://github.com/elastic/apm-server/issues/10292
